### ISSUE TITLE
Feature request: align Nostalgy default folder path on the left

### ISSUE THREAD

**Issue by mcacciari** [opto/nostalgy-xpi#75]
There's an old entry in the repository for the old Nostalgy, https://github.com/nostalgy/nostalgy/issues/87

The default folder path that Nostalgy displays is presently aligned on the right in the bottom bar. It would be much easier to read if it were on the left, to avoid having to look diagonally across the screen (since the list of messages is usually on the left)

Thunderbird could just hide occasionally Nostalgy content when needed (e.g. when hovering over a link)

If this is feasible, to have this at least as an option would be, by me, greatly appreciated.

**Comment by vvs** [comment]
Yep, a very desirable feature for me as well. The screens these days get very wide and looking at the bottom right corner every time I need to copy/move my emails is not optimal (at least for me). I'd much rather prefer the suggestions go to the left or even better in the middle. :)

Maybe there is a way to configure the position depending on the user preferences?

**Comment by opto** [comment]
I am not sure what you mean but develop on a notebook, so maybe cannot see that.

Here, I have the network indicator leftmost, next is the N++ help button, then left oriented the folder box. Is any of that right oriented in the most recent version? I think I changed that already. Which TB, N++ version and OS?

**Comment by mcacciari** [comment]
See below
![Screenshot 2021-02-01 at 22 29 01](https://user-images.githubusercontent.com/7535981/106521730-f3c19700-64de-11eb-8858-36cfe657e13d.jpg)
 the screenshot of my Thunderbird window. As you can see, the

[+Shift: => .... default save path...] 

is on the right. In the old version it was on the left, right below the list of messages.

Admittedly, after a few weeks I almost got used to the new position. But if it's possible to add an option to align it on the left, below the list of messages,  without too much work, I'd appreciate.

I'm now on Nostalgy 2.2.4, Thunderbird 78.6.0, on MacOS Catalina 10.15.7

**Comment by Olifair** [comment]
I see exactly the same as @mcacciari on Windows 10, TB 78.7.0 Nostalgy++ 2.2.4 : network indicator in left, then there is a big empty space, then all right oriented N++Help | Unread|total|calendar
upon tapping S or G I get the input box between the N++Help and the Unread, but all is still right oriented. 
I personally do not find this a problem at all, actually I quite like it this way,  but just to confirm

**Comment by opto** [comment]
there is a huge/wide empty label to the left of the N++ button. Here, it sometimes flashes: nothing to download.

I could try to hide it when we display the folderbox. Maybe make it a pref so Olifair can keep it or the others can disable it (pref is less work than a visible  option field in the UI).
